JavaScript/React.md

```diff
@@ -313,6 +313,16 @@ render() {
 }
 ```
 
+### Conditional Rendering
+When you want to render either something or nothing, you can use the && operator.
+```
+// bad
+{condition ? (<span />) : null}
+
+// good
+{condition && (<span />)}
```

Comment:
Also need to add a comment that you need to be sure the condition returns a boolean or weird things could happen.